R/importMetaphlan.R

```diff
@@ -3,19 +3,25 @@
 #' @param file a single \code{character} value defining the file
 #'   path of the Metaphlan file. The file must be in merged Metaphlan format.
 #'
-#' @param colData a DataFrame-like object that includes sample names in
+#' @param col.data a DataFrame-like object that includes sample names in
 #'   rownames, or a single \code{character} value defining the file
 #'   path of the sample metadata file. The file must be in \code{tsv} format
-#'   (default: \code{colData = NULL}).
+#'   (default: \code{col.data = NULL}).
+#'   
+#' @param colData Deprecated. use \code{col.data} instead.
 #' 
-#' @param sample_meta a DataFrame-like object that includes sample names in
+#' @param col.data a DataFrame-like object that includes sample names in
 #'   rownames, or a single \code{character} value defining the file
 #'   path of the sample metadata file. The file must be in \code{tsv} format
-#'   (default: \code{sample_meta = NULL}).
+#'   (default: \code{col.data = NULL}).
+#'   
+#' @param sample_meta Deprecated. Use \code{col.data} instead.
 #' 
-#' @param phy_tree a single \code{character} value defining the file
+#' @param phy.tree.file a single \code{character} value defining the file
```

Comment:
file suffix was added to denote that this is a file. Not object.

Reply:
ACK.